CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# dbt_netsuite v0.19.0
+
```

Comment:
```suggestion
For Netsuite2, [PR #160](https://github.com/fivetran/dbt_netsuite/pull/160) includes the following updates: 
```

Reply:
Thanks for catching. Will update in the next commit.